lib/commands/test-init.ts

```diff
@@ -37,6 +37,25 @@ class TestInitCommand implements ICommand {
 				'save-dev': true,
 				optional: false,
 			});
+
+			let modulePath = path.join(projectDir, "node_modules", mod);
+			let modulePackageJsonPath = path.join(modulePath, "package.json");
+			let modulePackageJsonContent = this.$fs.readJson(modulePackageJsonPath);
+			let modulePeerDependencies = modulePackageJsonContent.peerDependencies || {};
+
+			for (let peerDependency in modulePeerDependencies) {
+				let dependencyVersion = modulePeerDependencies[peerDependency] || "*";
+
+				// catch errors when a peerDependency is already installed
+				// e.g karma is installed; karma-jasmine depends on karma and will try to install it again
```

Comment:
The node-package-manager's install() that I used will fail the build - https://github.com/NativeScript/nativescript-cli/blob/master/lib/node-package-manager.ts#L78

I am unaware why that check is performed, even though the npm package is installed anyway at an earlier point - https://github.com/NativeScript/nativescript-cli/blob/master/lib/node-package-manager.ts#L42.

@rosen-vladimirov can you say what the reasons for that are?